inference-engine/src/transformations/src/transformations/common_optimizations/optimize_strided_slice.cpp

```diff
@@ -247,6 +391,10 @@ bool ngraph::pass::GroupedStridedSliceOptimizer::run_on_function(std::shared_ptr
 
 bool ngraph::pass::StridedSliceOptimization::run_on_function(std::shared_ptr<ngraph::Function> f) {
     RUN_ON_FUNCTION_SCOPE(StridedSliceOptimization);
+    ngraph::pass::Manager manager(get_pass_config());
+    manager.register_pass<ngraph::pass::SliceToStridedSlice>();
```

Comment:
(Update) I've moved the `SliceToStridedSlice` transformation code to the separate file in `op_conversions` dir.

But the right place to register `pass::SliceToStridedSlice>()` still need confirmation 
(@GlebKazantaev @itikhono).

Reply:
You can keep source code in op_conversion catalog since this transformation convert one operation to another. But the right place to call this transformation is still StridedSliceOptimization pass.

Reply:
So are you ok with the proposed implementation?

Reply:
Discussed to keep it as is.